components/git/security.js

```diff
@@ -17,17 +22,31 @@ export function builder(yargs) {
   yargsInstance = yargs;
   return yargs.options(securityOptions).example(
     'git node security --start',
-    'Prepare a security release of Node.js');
+    'Prepare a security release of Node.js')
+    .example(
+      'git node security --finalize',
+      'Finalize the date of the security release of Node.js'
```

Comment:
The name doesn't sound correct. We should make it more clear, like: `git node security --update=date`

To make it applicable for future updates, we should make it generic enough.